odev/common/actions.py

```diff
@@ -72,15 +73,15 @@ def _action_name(cls) -> str:
 class IntAction(Action):
     """Converter for command line arguments passed as a string that should be converted to an int."""
 
-    def _transform_one(self, value: str) -> int:
-        return int(value)
+    def _transform_one(self, value: Union[str, int]) -> int:
+        return int(value) if isinstance(value, str) else value
 
 
 class ListAction(Action):
     """Converter for command line arguments passed as comma-separated lists of values."""
 
     def _transform_one(self, value: str) -> List[str]:
-        return value.split(",")
+        return value.split(",") if isinstance(value, str) else value
```

Comment:
incoherent with the type hinting of the function, value is always a string according to the type hinting